ts/IterableInbox.tsx

```diff
@@ -1,13 +1,14 @@
 'use strict'
 
-import React, { useState, useEffect } from 'react'
+import React, { useState, useEffect, useCallback } from 'react'
```

Comment:
We don't seem to be using `useCallback`, should this be removed?

Reply:
Agreed. I will remove useCallback.